back-end/src/main/java/app/cluttermap/service/ItemsService.java

```
@@ -50,6 +50,9 @@ public Item updateItem(Long id, UpdateItemDTO itemDTO) {
         if (itemDTO.getDescription() != null) {
             _item.setDescription(itemDTO.getDescription());
         }
+        if (itemDTO.getQuantity() != null) {
```

You should add something to make sure the quantity is more than 0

If it is less than 0 you need to throw an exception